src/apps/competitions/tests/test_submissions.py

```diff
@@ -213,3 +216,29 @@ def test_making_submission_to_phase_with_one_task_does_not_create_parents_or_chi
         assert resp.call_count == 1
         sub = Submission.objects.get(id=self.sub.id)
         assert not sub.has_children
+
+    def test_adding_task_to_phase_runs_submissions_on_new_task(self):
+
+        leaderboard = LeaderboardFactory()
+
+        for phase in self.comp.phases.all():
+            phase.leaderboard = leaderboard
+            phase.save()
+
+        SubmissionFactory(owner=self.user, phase=self.phase)
+
+        competition_data = CompetitionSerializer(self.comp).data
+        new_task = TaskFactory()
+        competition_data["phases"][0]['tasks'].append(new_task.key)
+        competition_data['logo'] = None
+
+        for task_d, task in enumerate(competition_data["phases"][0]['tasks']):
```

Comment:
`task_d` is this a weird name, or am I not understanding something?

Reply:
Supposed to be task_*id*

Reply:
probably `task_index` makes sense right? I understand that it's annoying that I'm pressing on this.